cs/cs/Serializer/VowpalWabbitJsonSerializer.cs

```diff
@@ -102,6 +102,47 @@ public void Parse(string json, ILabel label = null, int? index = null)
             }
         }
 
+        public static int GetNumberOfActionDependentExamples(string json)
+        {
+            using (var textReader = new JsonTextReader(new StringReader(json)))
+            {
+                return GetNumberOfActionDependentExamples(textReader);
+            }
+        }
+
+        public static int GetNumberOfActionDependentExamples(JsonReader reader)
+        {
+            // handle the case when the reader is already positioned at JsonToken.StartObject
+            if (reader.TokenType == JsonToken.None && !reader.Read())
+                throw new VowpalWabbitJsonException(reader.Path, "Expected non-empty JSON");
+
+            if (reader.TokenType != JsonToken.StartObject)
+                throw new VowpalWabbitJsonException(reader.Path, "Expected start object");
+
+            while (reader.Read())
+            {
+                if (!(reader.TokenType == JsonToken.PropertyName && (string)reader.Value == "_multi"))
```

Comment:
Hmm. Actually that’s the string I wanted to make configurable. What do you think? Add as settings?